splitclient-rb.gemspec

```diff
@@ -46,6 +46,8 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'simplecov'
   spec.add_development_dependency 'timecop'
   spec.add_development_dependency 'webmock'
+  spec.add_development_dependency 'appraisal'
+  spec.add_development_dependency 'byebug'
```

Comment:
@we4tech First and foremost: **thanks so much for your contribution**, we really appreciate it! That said, we can tackle these comments after merging to our development branch, so you don't have to invest more time in this (unless, of course, you want to to tackle them yourself). Just let us know your preference, and we'll act accordingly.

Reply:
Really appreciate if you can take it from here @ezesplit